controls.py

```diff
@@ -148,12 +148,12 @@ def Sentence(*elems):
         groups.extend(elems[0])
 
     for i in range(start, len(elems) - (1 if tail else 0), 2):
-        assert (isinstance(elems[i], str) or isinstance(elems[i], html.Span))
+        #assert (isinstance(elems[i], str) or isinstance(elems[i], html.Span))
         groups.append(dbc.Label(elems[i], size="sm", className="m-1 d-inline-block", width="auto"))
         groups.extend(elems[i + 1])
 
     if tail:
-        assert (isinstance(elems[-1], str) or isinstance(elems[-1], html.Span))
+        #assert (isinstance(elems[-1], str) or isinstance(elems[-1], html.Span))
```

Comment:
This assertion says that we're not allowed to have two controls in a row, controls have to alternate with text; but the `DateNoYear` function generates two controls in a row: day of month and month. I'm not sure why this constraint was imposed, but it seems to have something to do with how groups are formed, which I'm not entirely following. @kgraaf please comment.

Reply:
More specifically, the problem was that DateNoYear was followed by Number. I interjected a "" inbetween the 2 and re-enabled the assertions and going through.